fido-device-onboard.spec

```diff
@@ -0,0 +1,122 @@
+%define dracutlibdir %{_prefix}/lib/dracut
+%bcond_without check
+%global __cargo_skip_build 0
+%global __cargo_is_lib() false
+%global gitversion 43f7674dad4abc470cdab505fbf49f571539cea8
+
+Name:           fido-device-onboard
+Version:        0.1.0
+Release:        1%{?dist}
+Summary:        An implementation of the FIDO Device Onboard Specification written in rust
+
+License:        BSD 3
+URL:            https://github.com/fedora-iot/fido-device-onboard-rs/
+Source:         %{url}/archive/%{gitversion}/%{name}-rs-%{gitversion}.tar.gz
+Source1:        %{name}-rs-%{gitversion}-vendor.tar.gz
+
+ExclusiveArch:  %{rust_arches}
+# RHBZ 1869980
+ExcludeArch:    s390x i686 %{power64}
+
+BuildRequires: rust-toolset
+BuildRequires: systemd
+
+%description
+%{summary}.
+
+%prep
+%autosetup -n %{name}-rs-%{gitversion} -p1
+%cargo_prep -V 1
+
+%build
+%cargo_build
+
+%install
+install -D -m 0755 -t %{buildroot}%{_libexecdir}/fdo target/release/fdo-client-linuxapp
+install -D -m 0755 -t %{buildroot}%{_libexecdir}/fdo target/release/fdo-manufacturing-client
+install -D -m 0755 -t %{buildroot}%{_libexecdir}/fdo target/release/fdo-manufacturing-server
+install -D -m 0755 -t %{buildroot}%{_libexecdir}/fdo target/release/fdo-owner-onboarding-server
+install -D -m 0755 -t %{buildroot}%{_libexecdir}/fdo target/release/fdo-rendezvous-server
+# TODO(runcom): we may want to move this to {_bindir} so admins can use it directly
+install -D -m 0755 -t %{buildroot}%{_libexecdir}/fdo target/release/fdo-owner-tool
+install -D -m 0644 -t %{buildroot}%{_unitdir} examples/systemd/*
+install -D -m 0644 -t %{buildroot}%{_datadir}/fdo examples/config/*
```

Comment:
Shouldn't this be `%{_sysconfdir}` (i.e. `/etc`)?

Reply:
we said we wanted these config under `/usr/share` - I'm fine either way, we just have to decide before I change it again :smile:

Reply:
also, those are just examples, not even made for actually starting the services (as you need keys and other stuff anyway)

Reply:
Ah, right. In that case, maybe they should go under `%{_docdir}`?

Reply:
Yes

Reply:
so `%{_docdir} it is